src/astro/constants.py

```diff
@@ -62,3 +62,5 @@ class Database(Enum):
 
 # TODO: check how snowflake names these
 AppendConflictStrategy = Literal["append", "replace", "exception"]
+
+DBMergeConflictStrategy = Literal["ignore", "update"]
```

Comment:
None of the other Strategies has DB in the name. Why aren't we using `AppendConflictStrategy` (which was meant for the hybrid append-merge)?